web/src/lib/components/utilities-page/duplicates/duplicates-compare-control.svelte

```diff
@@ -0,0 +1,133 @@
+<script lang="ts">
+  import Button from '$lib/components/elements/buttons/button.svelte';
+  import Icon from '$lib/components/elements/icon.svelte';
+  import { locale } from '$lib/stores/preferences.store';
+  import { getAssetThumbnailUrl } from '$lib/utils';
+  import { asByteUnitString } from '$lib/utils/byte-units';
+  import { ThumbnailFormat, type AssetResponseDto, type DuplicateResponseDto, getAllAlbums } from '@immich/sdk';
+  import { mdiCheck, mdiTrashCanOutline } from '@mdi/js';
+  import { onMount } from 'svelte';
+
+  export let duplicate: DuplicateResponseDto;
+  export let onResolve: (trashIds: string[]) => void;
+
+  let selectedAsset = new Set<string>();
+  $: trashCount = duplicate ? duplicate.assets.length - selectedAsset.size : 0;
+
+  onMount(() => {
+    const suggestedAsset = duplicate.assets.sort(
+      (a, b) => b.exifInfo!.fileSizeInByte! - a.exifInfo!.fileSizeInByte!,
+    )[0];
+
+    selectedAsset.add(suggestedAsset.id);
+    selectedAsset = new Set(selectedAsset);
```

Comment:
What's the purpose of this?

Reply:
reactivity, to trigger rerender

Reply:
Isn't `selectedAsset = selectedAsset` sufficient for that? No need to create a new set?

Reply:
Yes, rightly so